src/utils/useHotspot.ts

```diff
@@ -484,9 +481,23 @@ const useHotspot = () => {
       isFree,
       hasSufficientBalance,
       totalStakingAmount,
+      totalStakingAmountDC,
+      totalStakingAmountUsd,
+      remainingFreeAsserts: remainingFreeAsserts(),
     }
   }
 
+  const remainingFreeAsserts = () => {
+    if (!connectedHotspotDetails.validOnboarding) {
```

Comment:
your other PR gets rid of `validOnboarding` right? might need to adjust this to check for onboarding record

Reply:
ok, yeah, good call. Rebasing right now.